<head> <link rel="shortcut icon" type="image/x-icon" href="favicon.ico"></head>


<style>
  a:link {
    color: #176764;
    background-color: transparent;
    text-decoration: underline;
  }
  a:visited {
    color: #55A9A7;
    background-color: transparent;
    text-decoration: underline;
  }
  a:hover {
    color: #176764;
    background-color: transparent;
    text-decoration: underline;
  }
  a:active {
    color: #176764;
    background-color: transparent;
    text-decoration: underline;
  }
  h2 {
    text-align: center; 
    font-style: none; 
    font-weight: bold; 
    font-family: Century Gothic, sans-serif;
  }
  
  
.button {
  background-color: #04AA6D;
  border: none;
  color: white;
  padding: 15px 26px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  font-weight: bold;
  margin: 4px 2px;
  cursor: pointer;
  margin: 0;
  position: relative;
  left: 50%;
  -ms-transform: translate(-50%, -50%);
  transform: translate(-50%, -50%);
}

</style>

## **The siRNA Design Algorithm Web Portal is Live!**




<a href="http://sirna-frontend.s3-website.us-east-2.amazonaws.com/search">
<input type="button" value="Design Therapeutic siRNAs" class="button">
</a>



## [GitHub](https://github.com/kmonopoli)  |  [LinkedIn](https://www.linkedin.com/in/kathrynmonopoli/)  |  [CV](downloads/cv_kathryn_monopoli.pdf)  |  [Contact](mailto:krmonopoli@gmail.com)  |  [Publications](#publications)

<a href="#publications">**Publications**</a>


<img src="/images/photo_3.jpg" alt="image" width="300" align="left" style="display: block;  margin-right: 5px; margin: 5px;" />

I am a Ph.D. candidate studying Bioinformatics in the [Quantitative and Computational Biosciences & Bioengineering](https://umassmed.edu/es/gsbs/current-students/computational-biomedical-sciences-and-engineering/) Joint Graduate Program at University of Massachusetts Medical School and Worcester Polytechnic Institute. I am advised by [Anastasia Khvorova](https://www.umassmed.edu/khvorovalab/) and [Dmitry Korkin](http://korkinlab.org). I transitioned from the bench where I studied membrane protein biophysics, to computational research where I am applying machine learning techniques to study small interfering RNAs (siRNAs).

My research bridges biology with computer science to advance therapeutic development and, more generally, expand our understanding of biological processes. My primary focus is on applying machine learning techniques to probe the siRNA gene silencing mechanism and develop siRNA sequence design algorithms. I am passionate about increasing access to advanced computational algorithms for life science researchers without computational training, through developing easy-to-use computational tools that do not require any computer programming knowledge. 

I conducted my undergraduate studies at the Colorado School of Mines and the University of Massachusetts Amherst where I completed my B.S. *summa cum laude*. I completed my M.S. in Biochemistry in the Laboratory of [Alejandro Heuck](https://people.biochem.umass.edu/heuck/research.html) at the University of Massachusetts Amherst. 


# Experience
**Ph.D. Candidate** – Program in Quantitative and Computational Biosciences & Bioengineering  
&nbsp;&nbsp;&nbsp;Advised by Anastasia Khvorova & Dmitry Korkin  
&nbsp;&nbsp;&nbsp;University of Massachusetts Medical School & Worcester Polytechnic Institute  
&nbsp;&nbsp;&nbsp;RNA Therapeutics Institute & Bioinformatics and Computational Biology Department  

**Asociate Scientist Bioinformatics** – Advirna

**M.S. Graduate Researcher** – Biochemistry  
&nbsp;&nbsp;&nbsp;Advised by Alejandro Heuck  
&nbsp;&nbsp;&nbsp;University of Massachusetts Amherst  

**Intern** – Antibody Discovery  
&nbsp;&nbsp;&nbsp;Biogen  




# Publications
<div id="publications"></div>

**Monopoli KR**, Sostek B, Gross K, Alterman J, Korkin D, Khvorova A. (2024) MODsiGNer: fully modified siRNA design application applying artificial intelligence with deep sequence feature embedding. *In preparation for submission to Nature Methods.*

Weiss A, Gilbert JW, Flores IVR, Belgrad J, Ferguson C, Dogan EO, Wightman N, Mocarski K, Echeverria D, Summers A, Bramato B, McHugh N, Furgal R, Yamada N, Cooper D, **Monopoli KR**, Godinho BMDC, Hassler MR, Yamada K, Greer P, Henninger N, Brown RH Jr, Khvorova A. (2024) [RNAi-mediated silencing of SOD1 profoundly extends survival and functional outcomes in ALS mice](https://www.biorxiv.org/content/10.1101/2024.06.20.599943v1). *bioRxiv*. 

Rivera Flores IV, **Monopoli KR**, Jackson S, Echeverria D, O'Reilly D, Brown RH, Khvorova A. (2024) [Near Sequence Homology Does Not Guarantee siRNA Cross-Species Efficacy](https://www.liebertpub.com/doi/10.1089/nat.2024.0030?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed). *Nucleic Acid Therapeutics*.

Tang Q, Gross KY, Fakih HH, Jackson SO, UI Abideen MZ, **Monopoli KR**, Blanchard C, Bouix-Peter C, Portal T, Harris JE, Khvorova A, Alterman JF. (2024) [Multispecies-targeting siRNAs for the modulation of JAK1 in the skin](https://www.cell.com/molecular-therapy-family/nucleic-acids/fulltext/S2162-2531(24)00004-0). *Molecular Therapy Nucleic Acids*.

**Monopoli KR**, Korkin D, Khvorova A. (2023) [Asymmetric trichotomous data partitioning enables development of predictive machine learning models using limited siRNA efficacy datasets](https://www.cell.com/molecular-therapy-family/nucleic-acids/pdf/S2162-2531(23)00160-9.pdf?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2162253123001609%3Fshowall%3Dtrue). *Molecular Therapy Nucleic Acids*.

Davis SM, Hildebrand S, **Monopoli KR**, MacMillan H, Sousa J, Cooper D, Ly Socheata; Echeverria D, et al. (2023) Guidelines for Designing Therapeutic siRNA. *Under review with NAR.*

Hariharan VN, Shin M, Chang CW, O’Reilly D, Biscans A, Yamada K, Guo Z, Somasundaran M, Tang Q, **Monopoli KR**, Krishnamurthy PM, Devi G, McHugh N, Cooper DA, Echeverria D, Cruz J, Chan IL, Liu P, Lim SY, McConnell J, Singh SP, Hildebrand S, Sousa J, Davis SM, Kennedy Z, Ferguson C, Godinho BMDC, Thillier Y, Caiazzi J, Ly S, Muhuri M, Kelly K, Humphries F, Cousineau A, Parsi KM, Li Q, Wang Y, Maehr R, Gao G, Korkin D, McDougall WM, Finberg RW, Fitzgerald KA, Wang JP, Watts JK, Khvorova A. (2023) [Divalent siRNAs are bioavailable in the lung and efficiently block SARS-CoV-2 infection](https://www.pnas.org/doi/10.1073/pnas.2219523120). *PNAS*. 

O'Reilly D, Belgrad J, Ferguson C, Summers A, Sapp E, McHugh C, Mathews E, Boudi A, Buchwald J, Ly S, Moreno D, Furgal R, Luu E, Kennedy Z, Hariharan V, **Monopoli KR**, Yang XW, Carroll J, DiFiglia M, Aronin N, Khvorova A. (2023) [Divalent siRNA Mediated Silencing of MSH3 Blocks Somatic Repeat Expansion in Mouse Models of Huntington’s Disease](https://www.cell.com/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(23)00503-8). *Molecular Therapy.*

Tang Q, Fakih HH, Zain Ui Abideen M, Hildebrand SR, Afshari K, Gross KY, Sousa J, Maebius AS, Bartholdy C, Søgaard PP, Jackerott M, Hariharan V, Summers A, Fan X, Okamura K, **Monopoli KR**, Cooper DA, Echeverria D, Bramato B, McHugh N, Furgal RC, Dresser K, Winter SJ, Biscans A, Chuprin J, Haddadi NS, Sherman S, Yıldız-Altay Ü, Rashighi M, Richmond JM, Bouix-Peter C, Blanchard C, Clauss A, Alterman JF, Khvorova A, Harris JE. (2023) [Rational design of a JAK1-selective siRNA inhibitor for the modulation of autoimmunity in the skin](https://www.nature.com/articles/s41467-023-42714-4). *Nature Communications.*

Tang Q, Sousa J, Echeverria D, Fan X, Hsueah YC, Afshari K, MeHugh N, Cooper DA, Vangjeli L, **Monopoli KR**, et al. (2022) [RNAi-based modulation of IFN-γ signaling in skin](https://www.cell.com/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(22)00249-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1525001622002490%3Fshowall%3Dtrue). *Molecular Therapy.*

Shmushkovich, T.,\* **Monopoli, K.R.,\*** Homsy, D., Leyfer, D., Betancur-Boissel, M., Khvorova, A., Wolfson, A. (2018) [Functional features defining the efficacy of cholesterol-conjugated, self-deliverable, chemically modified siRNAs](https://academic.oup.com/nar/article/46/20/10905/5085976). *Nucleic Acids Research.* \***equally-contributing first authors** 

Romano FB, Rossi KC, Tang Y, Monopoli KR, Ross JL, Heuck AP. (2016) [Type 3 Secretion translocators spontaneously assemble a hexadecameric transmembrane complex](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995524/). *JBC.*

*[[ORCID: 0000-0002-8615-4849]](https://orcid.org/0000-0002-8615-4849)*


# Talks
Monopoli, K.R., Korkin, D., Khvorova, A. Trichotomous classification on small, limited datasets enables predictive model development for therapeutic small interfering RNA. Talk presented at the Intelligent Systems for Molecular Biology (ISMB); 2022 Jul 10; Madison, WI.[[recorded talk]](https://youtu.be/V1cT0HU0lJY)

Monopoli, K.R., Korkin, D., Khvorova, A. Evaluation-centric method for extracting base preferences from siRNA prediction models identifies features consistent with established mechanisms and is adaptable to examine any machine learning model. Talk presented at the RNA Therapeutics Symposium; 2022 Jun 22; Worcester, MA. [[recorded talk]](https://youtu.be/f_4O-Cq9PG0)

Monopoli, K.R., Korkin, D., Khvorova, A.  Methods to apply and evaluate machine learning models on limited biological datasets through the lens of siRNA design. Invited talk presented at Oligonucleotide Therapeutics Society Seminar; 2021 Oct.

# Conference Posters
Monopoli, K.R., Korkin, D., Khvorova, A. Artificial intelligence-based nonlinear sequence environment encapsulation yields robust native context therapeutic siRNA potency prediction algorithm. Presented at the Oligonucleotide Therapeutics Society Annual Meeting; 2023 Oct 23; Barcelona, Spain.

Monopoli, K.R., Korkin, D., Khvorova, A. Developing accurate models for computing potency of therapeutically modified siRNAs using AI-based methods to encapsulate sequence environments. Presented at the Nature Conference - Cracking the Code: The Dawn of Nucleic Acid Medicines; 2023 Oct 19; Worcester, MA.

Monopoli, K.R., Korkin, D., Khvorova, A. Method for encapsulating transcript sequence environment information boosts siRNA potency prediction accuracy of supervised machine learning models. Presented at the RNA Therapeutics Symposium; 2023 Jun 23; Worcester, MA.

Monopoli, K.R., Korkin, D., Khvorova, A. Trichotomous classification on small, limited datasets enables predictive model development for therapeutic small interfering RNA. Presented at the Intelligent Systems for Molecular Biology (ISMB); 2022 Jul 10; Madison, WI. [[recorded poster presentation]](https://youtu.be/qz2ApDI7DBI)

Monopoli, K.R., Korkin, D., Khvorova, A. Data Partitioning to Enable Application of Machine Learning Models to Limited Biological Datasets Using SiRNA Design as an Example Case. Presented at the Oligonucleotide Therapeutics Society Annual Meeting; Sep 2021; virtual. [[recorded poster presentation]](https://youtu.be/xtk9x210RmE)

Monopoli, K.R., siRNA Screening Consortium, Korkin, D., Khvorova, A. Predicting siRNA Silencing Efficacy using Supervised Machine Learning. Presented at the RNA Therapeutics Symposium; 2021 June 28; Worcester, MA. [[poster]](https://vepimg.b8cdn.com/uploads/vjfnew/4489/content/images/1622548685rti-poster-kathryn-monopoli-pdf1622548685.pdf)[[video overview]](https://vimeo.com/557558254)

Monopoli, K.R., Heuck, A.P. Forming the Pseudomonas aeruginosa translocon requires simultaneous incorporation of PopB and PopD. Presented at the Biophysical Society Annual Meeting; 2015 Feb 6-11; Baltimore, MD. [[abstract]](https://www.cell.com/biophysj/comments/S0006-3495(14)01752-4)

Monopoli, K.R., Romano, F.B., Heuck, A.P. Structural Studies of the Membrane Bound Translocon of the Pseudomonas aeruginosa Type III Secretion System. Poster presented at the Undergraduate Symposium for the American Chemical Society; 2014 April 26; Amherst, MA.

Monopoli, K.R., Romano, F.B., Heuck, A.P. Characterization of the Pseudomonas aeruginosa Type III Secretion Translocon in Model Membranes. Poster presented at the Massachusetts Undergraduate Statewide Research Conference; 2014 April 25; Amherst, MA.

Monopoli, K.R., Pearse, B.R. Generation of a repertoire of antibodies for clinical assay development by phage display. Poster presented at Biogen Intern Poster Session; 2013 Aug 22; Cambridge, MA.  

## Theses
Monopoli, K.R. Advised by Prof. Alejandro Heuck. “Characterization of the Reconstituted and Native Pseudomonas aeruginosa Type III Secretion System Translocon.” Master's Thesis. University of Massachusetts, Amherst, September 2015. 

Monopoli, K.R. Advised by Prof. Alejandro Heuck. “Characterization of the Pseudomonas aeruginosa Type III Secretion System Translocon in Model Membranes.” Undergraduate Thesis. University of Massachusetts, Amherst, May 2014. 

## Patents and Applications
Oligonucleotides for MAPT Modulation – US Utility Patent Application No. 62/991,405, 3/18/2020  
Oligonucleotides for SNCA Modulation – US Utility Patent Application No. 62/991,406, 3/18/2020  
Oligonucleotides for MSH3 Modulation – US Patent Application No. 63/012,603, 4/20/2020  
Oligonucleotides for MAPT Modulation – US Utility Patent Application No. 63/071,106, 8/27/2020  
Oligonucleotides for SNCA Modulation – US Utility Patent Application No. 63/071,115, 8/27/2020  
Oligonucleotides for SARS-CoV-2 Modulation – US Utility Patent Application No. 63/031,222, 5/28/2020  
Oligonucleotides for SARS-CoV-2 Modulation – US Utility Patent Application No. 63/084,817, 9/29/2020 [[view]](https://image-ppubs.uspto.gov/dirsearch-public/print/downloadPdf/20230021431)  
Oligonucleotides for MAPT Modulation – US Patent Application No. 17/204,480, 3/17/2021  
Oligonucleotides for MAPT Modulation – Intl Patent Application No. PCT/US2021/022688, 3/17/2021  
Oligonucleotides for SNCA Modulation – US Patent Application No. 17/204,483, 3/17/2021  
Oligonucleotides for SNCA Modulation – Intl Patent Application No. PCT/US2021/022748, 3/17/2021  
Oligonucleotides for SARS-CoV-2 Modulation – US Patent Application No. 17/333,839, 5/28/2021  
Oligonucleotides for SARS-CoV-2 Modulation – Intl Patent App No. PCT/US2021/035002, 5/28/2021  

## Teaching Experience
CS 2102: **Advanced Object-Oriented Design Concepts** (WPI) – Teaching Assistant (Fall 2019)  
CS 1101: **Introduction to Programming Design** (WPI) – Teaching Assistant (Fall 2019)  
BIOCHEM 471: **Physical Chemistry** (UMass Amherst) – Teaching Assistant(Spring 2014)  
BIOLOGY 311: **General Genetics** (UMass Amherst) – Teaching Assistant (Fall 2013)  



## Fellowships and Awards
Travel Fellowship - Oligonucleotide Therapeutics Society 2023 Annual Meeting 
iRNA COSI Travel Fellowship - ISMB 2022 Conference  
Poster Award – Oligonucleotide Therapeutics Society 2021 Annual Meeting  
Fuller Scholarship  
Phi Beta Kappa  
Kappa Mu Epsilon – Honors Society for Mathematics  
Henry Little Award for Excellence in Research and Academics  
GO-MAP Graduate Top Scholars Award  
Biophyscal Society Travel Award  
Honors Research Grant – University of Massachusetts Amherst  
Research Assistant Fellowship – University of Massachusetts Amherst  
Phi Kappa Phi  
Presidential Scholarship – Colorado School of Mines  

## Service
[TouchTomorrow WPI](https://wp.wpi.edu/touchtomorrow/) - volunteer  
[UMass STEM Ambassadors](https://www.umass.edu/stemambassadors/mentoring-opportunities) - mentor  
[Biogen Community Lab](https://www.biogen.com/en_us/community-lab-summer.html) - instructor  
[Engineering Projects in Community Service](https://minesmagazine.com/2482/) - project manager, designer  
[Bright Spot Therapy Dogs](https://www.umass.edu/stemambassadors/mentoring-opportunities) - handler  
[Hector Reyes House](https://www.lahaworc.org) - volunteer  







